edu_edfi_airflow/dags/edfi_resource_dag.py

```diff
@@ -245,6 +257,36 @@ def chain_task_groups_into_dag(self):
         else:
             raise ValueError(f"Run type {self.run_type} is not one of the expected values: [default, dynamic, bulk].")
 
+        # Set up DAG-wide bearer token manager
+        # for default/dynamic dags only
+
+        if self.use_shared_edfi_token_provider:
```

Comment:
Is there a world where the `EdFiTokenProvider` and the `ExternalTaskSensor` are the same operator?